loop/config.js

```diff
@@ -529,6 +529,17 @@ var conf = convict({
       default: "",
       env: "ROOMS_HKDF_SECRET"
     }
+  },
+  ga: {
+    activated: {
+      doc: "Defines if uncaught exceptions should dump snapshots",
```

Comment:
I don't think this is the right doc?  I think it should be like "If true then POST /events will send event information to Google Analytics"